sympy/physics/vector/functions.py

```diff
@@ -592,15 +592,23 @@ def dynamicsymbols(names, level=0):
     ========
 
     >>> from sympy.physics.vector import dynamicsymbols
-    >>> from sympy import diff, Symbol
+    >>> from sympy import diff, Symbol, sqrt
     >>> q1 = dynamicsymbols('q1')
     >>> q1
     q1(t)
+    >>> q2= dynamicsymbols('q2', real=True)
+    q2(t)
+    >>> q2.is_real
+    True
+    >>> q3= dynamicsymbols('q3', positive=True)
+    q3(t)
+    >>> q3.is_positive
+    True
```

Comment:
This is good. Also add a note in the Parameters section explaining what `**asumptions` is. You can reference the `symbols()` or `Symbol()` documentation.